process/tilt/post-ac-review.html

```diff
@@ -0,0 +1,134 @@
+    7-10 days to express dissent.</li>
+  <li>In parallel or subsequently, the charter shepherd also
+    <a href="#TiLT-report">reports</a> the review results to TiLT. That message
+    should highlight the same (7-10 day) window for TiLT to request discussion
+    of the changes.</li>
+</ul>
+
+<p>If the time period elapses without dissent by AC Reps and TiLT gave its
+  <a href="./#timing">approval</a>,
+  the charter shepherd proceeds directly to 
+<a href="../charter.html#director-decision">requesting
+  announcement of the charter</a>. Otherwise, TiLT discussion takes
+place. Upon subsequent approval, the charter shepherd
+<a href="../charter.html#director-decision">requests announcement</a>.</p>
+
+<h3>Case 3: Uncontroversial Review but TiLT Approval Required</h3>
+
+<p>If there are no Formal Objections and no changes, then there are
+  still two scenarios when TiLT approval is required:</p>
+
+<ul>
+  <li>In the TiLT decision to request AC review, there was
+      a requirement to discuss the AC review results whatever the outcome
+      (e.g., to assign FTE based on the level Member support).</li>
+  <li>The review did not reach the 5% threshold and TiLT did not
+    relax that requirement in the decision to seek AC review.</li>
+</ul>
+
+<p>In either case, the charter shepherd <a href="#TiLT-report">reports</a>
+  the review results. That message should highlight that the review was
+  uncontroversial and point out which topics now require TiLT
+  approval. Upon subsequent approval, the charter shepherd
+<a href="../charter.html#director-decision">requests announcement of the charter</a>.</p>
+
+<h3>Case 4: Uncontroversial Review and No TiLT Approval Required</h3>
+
+<p>The charter shepherd <a href="#TiLT-report">reports</a> the review
+  results to TiLT and indicates that the charter is deemed approved and
+  no further TiLT approval is required.</p>
+
+<p>The charter shepherd then
+  <a href="../charter.html#director-decision">requests announcement of the charter</a>.</p>
+
+<h3 id="TiLT-report">Reporting AC Review Results to TiLT</h3>
+
+<p>In all of the above cases, the charter shepherd reports the review
+outcomes to <a
+href='https://github.com/w3c/tilt-private/issues/new/choose'>TiLT</a> with the
+following information:</p>
+
+<ul>
+<li>The distribution of reviews. It helps to highlight either that there have been
+  Formal Objections or that there is Consensus.</li>
+
+<li>A record of any substantial review comments and replies to those comments. In particular, for any Formal Objection, whether:
+  <ul>
+    <li>No attempt has been made to resolve it.</li>
+    <li>An attempt has been made to resolve it but it has remained unresolved
+      (e.g., because the reviewer did not respond, or no agreement can be reached).</li>
+    <li>It has been resolved. In this case indicate the resolution and whether
+      that involved changes to the charter.</li>
+    <li>It has been overridden by the Director (e.g, deemed out of scope or
```

Comment:
I'm not sure what is the correct wording, but `overridden by the Director` seems to be old sentence..

Reply:
Fixed